data/json/requirements/cooking_tools.json

```diff
@@ -18,6 +18,7 @@
     "tools": [
       [
         [ "hotplate", 1 ],
+		[ "multi_cooker", 1 ],
```

Comment:
Could you use spaces instead of tabs here? That's why it's showing out of alignment.

Reply:
@ZhilkinSerg haha looks like you beat me to it.

Reply:
> haha looks like you beat me to it.

 ⚔️ ✌️